pro/src/core/OfferEducational/adapters/getStockOfferAdapter.ts

```diff
@@ -0,0 +1,41 @@
+import { Offer } from 'custom_types/offer'
+import * as pcapi from 'repository/pcapi/pcapi'
+
+import { GetStockOfferSuccessPayload } from '../types'
+
+type IPayloadFailure = null
+type GetOfferAdapter = Adapter<
+  string,
+  GetStockOfferSuccessPayload,
+  IPayloadFailure
+>
+
+const FAILING_RESPONSE: AdapterFailure<IPayloadFailure> = {
+  isOk: false,
+  message: 'Une erreur est survenue lors de la récupération de votre offre',
+  payload: null,
+}
+
+export const getStockOfferAdapter: GetOfferAdapter = async offerId => {
+  try {
+    const offer = (await pcapi.loadOffer(offerId)) as Offer
+
+    /* @debt bugRisk "Gaël: we can't be sure this way that the stock is really booked, it can also be USED or CANCELED"*/
```

Comment:
(détail, commentaires) 

```suggestion
    /* @debt bugRisk "Gaël: we can't be sure this way that the stock is really booked, it can also be USED"*/
```

`bookingsQuantity` vient de `Booking.dnBookedQuantity` qui ne prend en compte que les réservations en cours ou validées. Les réservations annulées ne sont pas décomptées.